auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -180,44 +178,54 @@ public void checkAccountExists(@NonNull final String email) {
                     .fetchProvidersForEmail(email)
                     .addOnFailureListener(
                             new TaskFailureLogger(TAG, "Error fetching providers for email"))
-                    .addOnCompleteListener(getActivity(), new OnCompleteListener<ProviderQueryResult>() {
-                        @Override
-                        public void onComplete(@NonNull Task<ProviderQueryResult> task) {
-                            mHelper.dismissDialog();
-                        }
-                    })
-                    .addOnSuccessListener(getActivity(), new OnSuccessListener<ProviderQueryResult>() {
-                        @Override
-                        public void onSuccess(ProviderQueryResult result) {
-                            List<String> providers = result.getProviders();
-                            if (providers == null || providers.isEmpty()) {
-                                // Get name from SmartLock, if possible
-                                String name = null;
-                                Uri photoUri = null;
-                                if (mLastCredential != null && mLastCredential.getId().equals(email)) {
-                                    name = mLastCredential.getName();
-                                    photoUri = mLastCredential.getProfilePictureUri();
-                                }
-
-                                mListener.onNewUser(new User.Builder(email)
-                                                            .setName(name)
-                                                            .setPhotoUri(photoUri)
-                                                            .build());
-                            } else if (EmailAuthProvider.PROVIDER_ID.equalsIgnoreCase(providers.get(0))) {
-                                mListener.onExistingEmailUser(new User.Builder(email).build());
-                            } else {
-                                mListener.onExistingIdpUser(new User.Builder(email)
-                                                                    .setProvider(providers.get(0))
+                    .addOnCompleteListener(getActivity(),
```

Comment:
nit: the formatting between this complete listener and the success listener that follows is inconsistent.  I prefer any style that minimizes indentation which means I vote for either the style before this change or the success listener style below.